cmd/launcher/options.go

```diff
@@ -200,6 +216,8 @@ func shortUsage(flagset *flag.FlagSet) {
 	fmt.Fprintf(os.Stderr, "\n")
 	printOpt("version")
 	fmt.Fprintf(os.Stderr, "\n")
+	printOpt("osquery_flag")
```

Comment:
I think this belongs in `developerUsage` not `shortUsage`. I can be convinced otherwise, but my gut sense is pretty clear. I think this is advanced/uncommon usage.